packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -568,6 +574,17 @@ export const Table = <
     ],
   );
 
+  const scrollToIndex = React.useMemo((): number | undefined => {
+    if (!enableVirtualization || paginatorRenderer) {
+      return undefined;
+    }
+    const index = rows.findIndex((row) => row.original == scrollToItem);
```

Comment:
Hmm.. Could you add an example on storybook? Currently I am thinking of passing an item versus passing an index... 

Of course, need to think about filtered out list of items too, so not sure :/